packages/terra-navigation-side-menu/src/terra-dev-site/doc/example/NavigationSideMenuExample.scss

```diff
@@ -0,0 +1,15 @@
+:local {
+  .content {
+    background-color: #ffc0cb;
+    height: 100%;
+    position: relative;
+    width: 100%;
+  }
```

Comment:
We can import this from NavigationSideMenuDocCommon

Reply:
fixed here https://github.com/cerner/terra-framework/pull/693/commits/fe6b8b31fa9c34b77574262f68e1b3ed346dff49.